runtime/src/chpl-init.c

```diff
@@ -346,6 +350,14 @@ void chpl_std_module_init(void) {
 
 }
 
+void chpl_std_module_finalize(void) {
+  // TODO do we need this?
```

Comment:
`// TODO do we need this?`

Given that this isn't called anywhere I'm going to say no, we should just get rid of this.